### PR TITLE
Add EE 10 schemas from https://jakarta.ee/xml/ns/jakartaee/#10

### PR DESCRIPTION
**Pull request by hs536**
* follow-up: https://github.com/eclipse-ee4j/glassfish/pull/23816

Add the missing schema files for EE10 published from https://jakarta.ee/xml/ns/jakartaee/#10

Signed-off-by: hs536 <sawamura.hiroki@fujitsu.com>

### CODE REVIEW

**Comment by pzygielo** [comment]
It looks that committed files were re-formatted, as they differ between this PR and https://jakarta.ee/xml/ns/jakartaee/#10.
I'd prefer to have them copied 1:1.

**Comment by hs536** [comment]
@pzygielo Thanks for your comment. The preceding MR was formatted, so I went along with that, but would it be preferable to store the raw file as is?

**Comment by pzygielo** [comment]
> The preceding MR was formatted,

I have no time to complain about everything :grin: 

> would it be preferable to store the raw file as is just for this ones?

I only expressed my preference - to have no difference found between **files** published by Jakarta EE and stored by GlassFish. I used `diff`, and it was not happy.

I didn't compare **the contents**, which is the thing that matters in the end, so it might be OK - but I don't know.

**Comment by arjantijms** [comment]
As the previous one was formatted, let's do it here too. If it appears to be troublesome later, we can unformat them all?

**Review by arjantijms — APPROVED** [review]

**Review by dmatej — DISMISSED** [review]

**Comment by dmatej** [comment]
I would prefer even those original files would be without trailing spaces and well formatted ;)
Otherwise it is always an issue here and there. 
Some tools can live with that (kdiff3 in Kubuntu can compare even server.log files).

File: appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd [hunk 1]
```diff
@@ -0,0 +1,1163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (pzygielo) [inline appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd:4]: At https://jakarta.ee/xml/ns/jakartaee/#10 - it is `2009, 2021`.
So this isn't just copied and re-formatted.
What is the source of this file, then?

File: appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd [hunk 2]
```diff
@@ -0,0 +1,1163 @@
...
+	    targetNamespace="https://jakarta.ee/xml/ns/jakartaee"
+	    xmlns:jakartaee="https://jakarta.ee/xml/ns/jakartaee"
+	    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+	    elementFormDefault="qualified"
+	    attributeFormDefault="unqualified"
+	    version="2.1">
+
+  <xsd:annotation>
+    <xsd:documentation>
+      <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="license.inc" parse="text"/>
```
Comment (pzygielo) [inline appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd:30]: Where one can find this `license.inc`?

Comment (pzygielo) [reply appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd:30]: Just guessing - all files in this PR were copied without a change from https://github.com/eclipse-ee4j/jakartaee-schemas/tree/master/xml/src, from `xsds`es.
I'd expect them to be processed and taken as published instead.

File: appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd (hunk 1 above)
Comment (hs536) [reply appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd:4]: As you commented later, the source of this file is [the file below https://github.com/eclipse-ee4j/jakartaee-schemas/tree/master/xml/src.](https://github.com/eclipse-ee4j/jakartaee-schemas/blob/master/xml/src/connector_2_1.xsds).

File: appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd (hunk 2 above)
Comment (hs536) [reply appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/connector_2_1.xsd:30]: Thank you for the catch. I was not aware that the original file did not match the published file.
So... for now, I'll just format the currently published xsd file according to [Eclipse codestyle](https://github.com/eclipse-ee4j/ee4j/tree/master/codestyle).

**Review by pzygielo — APPROVED** [review]
Checked that new files in 7d8d1526bdeb817c926774d590c6e2cc214f3735 are the same as at https://jakarta.ee/xml/ns/jakartaee/#10.

**Review by dmatej — APPROVED** [review]

**Pull request merged**